Makefile

```diff
@@ -33,7 +46,8 @@ lint: $(TOOLS_DIR)/golangci-lint
 
 compile:
 	@echo "=== $(INTEGRATION) === [ compile ]: Building $(INTEGRATION)..."
-	@go build -o $(BUILD_TARGET) ./cmd/nri-kube-events
+	go mod download
```

Comment:
why we pre-download the modules ? Also just out of curiosity.

Reply:
Nice point, not really sure why I added it here.

Reply:
Ok I just remembered, it is needed if the build tree is in the GOPATH, as `go build` doesn't download modules automatically in that case.